### DISCUSSION TITLE
Estimating ambient profile from empty droplet matrix using DecontPro

### DISCUSSION THREAD

**Discussion by redwanfarooq** [campbio/decontX#14] [Q&A] [answered]
I'm trying to apply DecontPro to ADT data from CITE-seq and TEA-seq experiments. In your paper, you state:

> our software allows users to input the empty droplets to more accurately estimate the ambient profile

However, I'm unable to find the parameter in the `decontPro` function to input the empty droplet ADT count matrix. Could you please clarify how this functionality is implemented?

Many thanks in advance for your help!

**Comment by yuan-yin-truly** [comment, marked as answer]
Hey @redwanfarooq thank you for the great question! The short answer is that I have that workflow in the script I run experiments with when writing the paper, but forgot to add it when putting things together in the software package.

So I just added that to the `devel` branch. Could you try installing it and let me know if it works?
```
library(devtools)
devtools::install_github("campbio/decontX", ref = "devel")
```
The function documents are also updated. If you try `?decontPro` after installing, you should see that `decontPro()` has a new param called `ambient_counts`. It should be a count matrix containing only the ambient droplets, i.e. those with low RNA and ADTs.

Let me know if you encounter any other issues. Thanks!

**Reply by redwanfarooq** [reply]
Thanks for the quick response @yuan-yin-truly! I've re-installed the package from the development branch and it's all working fine. Are you planning to add this functionality to the Bioconductor package in due course?

**Reply by yuan-yin-truly** [reply]
Yes definitely. The `devel` branch will be the next release branch on Bioconductor. In the meantime, I will try to find some time to update the vignette as well.
